android-client-sdk/src/main/java/com/devcycle/sdk/android/model/PopulatedUser.kt

```diff
@@ -53,7 +50,7 @@ internal data class PopulatedUser constructor(
     val lastSeenDate: Long? = Calendar.getInstance().time.time,
 ) {
     @Throws(IllegalArgumentException::class)
-    @JvmSynthetic internal fun copyUserAndUpdateFromDVCUser(user: DVCUser): PopulatedUser {
+    @JvmSynthetic internal fun copyUserAndUpdateFromDVCUser(user: DevCycleUser): PopulatedUser {
```

Comment:
should we rename this to `copyUserAndUpdateFromDevCycleUser` or are we okay keeping it as is since its an internal function?